colormaps/core.py

```diff
@@ -280,7 +280,10 @@ def convert(self, data, limits):
             limits = self.limits
 
         data = self.process(data=data, limits=limits)
-        return self.rgba[np.uint64(len(self) * data)]
+
+        # Mask invalid data created by discretisation errors etc.
+        idx = np.abs(np.uint64(len(self) * data)).clip(0, len(self) + 1)
```

Comment:
- `np.abs` van een `uint` is overbodig

Ik denk dat je het best`idx = np.round(len(self) * data).astype(np.uint64)` kan doen, en dan de clip